### PR TITLE
Updated API endpoints and specs

### PR DESCRIPTION
**Pull request by ts-leojoseph-r**
# Description
Updated API endpoints and spec

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [x] I wrote/updated tests for new/changed code
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

File: Example/ViewController.swift [hunk 1]
```diff
@@ -30,8 +30,7 @@ class ViewController: UITableViewController {
         let cell = tableView.dequeueReusableCell(withIdentifier: "MiniAppCell", for: indexPath)
 
         let miniAppDetail = self.decodeResponse?[indexPath.row]
-        cell.textLabel?.text = miniAppDetail?.name
-        cell.detailTextLabel?.text = miniAppDetail?.description
```
Comment (ts-nitish-a-jain) [inline Example/ViewController.swift]: Will the detail text label auto-hide when a value is not assigned?

Comment (ts-leojoseph-r) [reply Example/ViewController.swift]: Not auto-hide, there will be no value assigned(empty string) as it is optional

Comment (ts-nitish-a-jain) [reply Example/ViewController.swift]: Okay. On the Android side, we just modified the visibility to reuse the layout for the future (but we made sure that description text is not visible, to reflect that there have been updates on the BE API). You might want to consider the same.

Comment (ts-leojoseph-r) [reply Example/ViewController.swift]: Pushed the changes now.

**Pull request merged**